src/components/TaskList/TaskList.jsx

```diff
@@ -1,20 +1,31 @@
 const React = require('react')
+const { connect } = require('react-redux')
+
+const Task = require('../Task')
 
 class TaskList extends React.Component {
+  onSnoozeClick() {
+    this.props.dispatch({ type: 'TASKS_SNOOZE' })
+  }
+
   render() {
     return (
       <div>
         <nav className="controls-container">
-          <button type="button" className="control">snooze</button>
+          <button id="snooze" type="button" onClick={() => this.onSnoozeClick()} className="control">snooze</button>
           <button type="button" className="control">ignore</button>
           <button type="button" className="control">archive</button>
         </nav>
         <ol className="issues-list">
-          <li>LOOK AT THIS ISSUE</li>
+          {this.props.tasks.map(task => {
+            return (<Task key={task.id} task={task} />)
+          })}
         </ol>
       </div>
     )
   }
 }
 
-module.exports = TaskList
+const mapStateToProps = tasks => ({ tasks })
+
+module.exports = connect(mapStateToProps)(TaskList)
```

Comment:
> After explaining that I realize I don't need to use mapStateToProps here, so I'm going to remove it.

I'm actually going to leave this as is for now since I can't figure out how to get the state out of the props without using `mapStateToProps`